README.md

```diff
@@ -52,9 +52,33 @@ Prometheus.Counter.inc_one (Metrics.builds_succeeded_total build_type)
 Applications can enable metric reporting using the `prometheus-app` opam package.
 This depends on cohttp and can serve the metrics collected above over HTTP.
 
+For example a server running on Linux will typically use the following code to 
```

Comment:
This doesn't flow very well. We were talking about metrics; now suddenly we're configuring logging?